ap/plotting/plotter.py

```diff
@@ -188,7 +182,7 @@ def plot_all(plot_config: dict, style_config: dict, ratio: bool = True) -> plt.F
 
     cms_label_kwargs = {
         "ax": ax,
-        "label": "Work in Progress",
+        "llabel": "Work in progress",
```

Comment:
typo

Reply:
I also had to look twice, but the version of mplhep in the venv uses `llabel` (for left label).